string first_name
string last_name
sensor_msgs/PointCloud2 pc1
sensor_msgs/PointCloud2 pc2
